runners/console/index.ts

```diff
@@ -247,6 +247,32 @@ export class Console extends Runner {
         return result;
     }
 
+    runDownloadFile(step: Step, command: Command): RunResult {
+        let result = new RunResult();
+        result.returnCode = 0;
+
+        let downloadUrl = command.parameters[0];
+        let installDir = path.join(this.getVariable(this.workspaceDirectory), command.parameters[1]);
+        this.createFolder(installDir, false);
+
+        if (command.parameters.length > 2) {
+            let name = command.parameters[2];
+            if(this.platform == ConsolePlatform.WINDOWS) {
+                this.executeCommandSync("powershell.exe \"Invoke-WebRequest -OutFile " + name + " '" + downloadUrl + "'\"", installDir, result);
+            } else {
+                this.executeCommandSync("wget -c \"" + downloadUrl + "\" -O " + name, installDir, result);
+            }
+        } else {
+            if(this.platform == ConsolePlatform.WINDOWS) {
+                this.executeCommandSync("powershell.exe \"Invoke-WebRequest '" + downloadUrl + "'\"", installDir, result);
+            } else {
+                this.executeCommandSync("wget -c \"" + downloadUrl + "\"", installDir, result);
```

Comment:
I think we can shorten this. For example save the command in a variable, add the output filename parameter if one is given and then pass this to executeCommandSync.

Reply:
Would that really make it shorter?

Reply:
You then have only one call to executeCommandSync.